lms/static/scripts/frontend_apps/components/OAuth2RedirectErrorApp.js

```diff
@@ -23,20 +23,32 @@ export default function OAuth2RedirectErrorApp({ location = window.location }) {
     OAuth2RedirectError: {
       authUrl = /** @type {string|null} */ (null),
       invalidScope = false,
+      errorCode = '',
```

Comment:
The default value of `errorCode` should be strictly `null` according to the definition of the `errorCode` field, and that's what the backend actually uses as the default value. Additionally if you know the backend is always going to set a particular field, as is the case here, you don't need to specify a default.

Reply:
Well spotted, updated without a default